Formula/tbb.rb

```diff
@@ -80,7 +82,9 @@ def install
     system ENV.cxx, "sum1-100.cpp", "--std=c++14", "-L#{lib}", "-ltbb", "-o", "sum1-100"
     assert_equal "5050", shell_output("./sum1-100").chomp
 
-    system Formula["python@3.9"].opt_bin/"python3", "-c", "import tbb"
+    python = Formula["python@3.10"].opt_bin/"python3"
+    ENV.prepend_path "PYTHONPATH", prefix/Language::Python.site_packages(python)
```

Comment:
Seems like you shouldn't need this if we're already using Homebrew `python@3.10`.